lib/components/tabs/tabview.coffee

```diff
@@ -152,6 +155,7 @@ module.exports = class KDTabView extends KDScrollView
     if shouldDetach
       @panes   = @panes.filter   (p)-> p isnt pane
       @handles = @handles.filter (h)-> h isnt handle
+      pane.isDetached = yes
```

Comment:
this shouldn't be the way, if you check `KDView::detach` we unset parent and you can use that info to understand if a pane is detached or not.